force-app/main/default/classes/NextStepBatch.cls

```diff
@@ -22,36 +22,39 @@ public with sharing class NextStepBatch implements Database.Batchable<sObject>{
 
     public Database.QueryLocator start(Database.BatchableContext bc){
         // Query for all accounts and return the query locator.
-        return null; // Update null with the query locator
+        return Database.getQueryLocator([SELECT Id, Next_Step__c FROM Account]); // Update null with the query locator
```

Comment:
Not 100% necessary but you can declare your batches stateful if you want the job to remember the records that were processed.